docker-compose-dev.yml

```diff
@@ -1,12 +1,26 @@
- version: '2'
- services:
-   web:
-     build: .
-     ports:
-      - "8100"
-     environment:
-       REDIS_ENDPOINT: redis://redis:6379/0
-     depends_on:
+version: '2'
+services:
+  web:
+    container_name: functions_app_dev
+    build: .
+    ports:
+      - "127.0.0.1:8100:8100"
```

Comment:
@heitorsilva  yes, the first one is the port inside the container. The second one is the port on host machine. Also, the 127.0.0.1 avoids requests from the network or malicious machines.

As an example about the question: I can load a rails server inside container at port 3000 and bind it on 8000 at host machine. Then it would be "127.0.0.1:3000:8000" at docker-compose file.